tasks/single-zone.yml

```diff
@@ -32,3 +32,34 @@
   with_dict: "{{ item.metadata | default({'SOA-EDIT': ''}) }}"
   loop_control:
     loop_var: meta
+
+- name: Upsert RRsets
+  command:
+    cmd: >
+      /usr/local/bin/pdns-auth-api-upsert-rrset "{{ pdns_auth_api_connect }}" "{{ pdns_auth_api_server }}"
+      "{{ item.name }}"
```

Comment:
why don't you use `loop_control` for this element? you could then use `zone.name` instead

Reply:
Because it works like this and more code menas more comlexity

Reply:
but it would also make it easier to understand the code